Sources/SwiftRex/CoreTypes/StoreImplementations/ReduxPipelineWrapper.swift

```diff
@@ -1,23 +1,35 @@
 import Foundation
 
 public struct ReduxPipelineWrapper<MiddlewareType: Middleware>: ActionHandler
-    where MiddlewareType.InputActionType == MiddlewareType.OutputActionType {
+    where MiddlewareType.InputActionType == MiddlewareType.OutputActionType
+{
     public typealias ActionType = MiddlewareType.InputActionType
     public typealias StateType = MiddlewareType.StateType
 
     private var onAction: (ActionType, ActionSource) -> Void
+    private let middleware: MiddlewareWrapper
+
+    private class MiddlewareWrapper {
+        let middleware: MiddlewareType
+
+        init(middleware: MiddlewareType) {
+            self.middleware = middleware
+        }
+    }
 
     public init(
         state: UnfailableReplayLastSubjectType<StateType>,
         reducer: Reducer<ActionType, StateType>,
-        middleware: MiddlewareType,
+        middleware m: MiddlewareType,
```

Comment:
SwiftLint is a bit unhappy with that "m"  variable name. To make things easier I'm gonna merge your PR and fix CI in the next one, ok?